core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -1303,9 +1296,10 @@ func TestLogPoller_DBErrorHandling(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 	require.NoError(t, lp.Start(ctx))
 	require.Eventually(t, func() bool {
-		return observedLogs.Len() >= 5
+		return observedLogs.Len() >= 3
```

Comment:
@DylanTinianov  can you point me to the changes in LP that resulted in this diff?

Reply:
It seemed that the sql logs in testing from pg could have added to this?